<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>621</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>vSongBook for Desktop</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/icons/appicon.ico</normaloff>../res/icons/appicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="SplitterMain">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Trebuchet MS</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="layoutLeft">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabLeft">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabSearch">
           <attribute name="title">
            <string>Search</string>
           </attribute>
           <layout class="QVBoxLayout" name="tabSearchLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLineEdit" name="txtSearch">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string>Search for a Song</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbSongbooks">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>20</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkSearchCriteria">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Search All Song Books</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="lstResults">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabHistory">
           <attribute name="title">
            <string>History</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <layout class="QVBoxLayout" name="layoutRight">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolBar" name="toolBar">
          <property name="windowTitle">
           <string>toolBar</string>
          </property>
          <addaction name="toolbarPresent"/>
          <addaction name="separator"/>
          <addaction name="toolbarEdit"/>
          <addaction name="separator"/>
          <addaction name="toolbarPrevious"/>
          <addaction name="separator"/>
          <addaction name="toolbarNext"/>
          <addaction name="separator"/>
          <addaction name="toolbarSmaller"/>
          <addaction name="separator"/>
          <addaction name="toolbarBigger"/>
          <addaction name="separator"/>
          <addaction name="toolbarBold"/>
          <addaction name="separator"/>
          <addaction name="toolbarSettings"/>
          <addaction name="separator"/>
          <addaction name="actionDarkMode"/>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtTitle">
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Song Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtContent">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Song Content</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtAlias">
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Alias</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuPrint"/>
    <addaction name="separator"/>
    <addaction name="menuExit"/>
   </widget>
   <widget class="QMenu" name="menuSongs">
    <property name="title">
     <string>Songs</string>
    </property>
    <addaction name="menuNewSong"/>
    <addaction name="menuEditSong"/>
    <addaction name="separator"/>
    <addaction name="menuBooks"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="menuPresent"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="menuManageSettings"/>
    <addaction name="separator"/>
    <addaction name="menuDarkMode"/>
    <addaction name="menuSearchAll"/>
    <addaction name="separator"/>
    <addaction name="menuResetSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuHowItWorks"/>
    <addaction name="menuContactUs"/>
    <addaction name="menuAboutThisApp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSongs"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="menuNewSong">
   <property name="text">
    <string>New Song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="menuEditSong">
   <property name="text">
    <string>Edit Song</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="menuBooks">
   <property name="text">
    <string>Songbooks</string>
   </property>
   <property name="shortcut">
    <string>Shift+B</string>
   </property>
  </action>
  <action name="menuPresent">
   <property name="text">
    <string>Present</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="menuManageSettings">
   <property name="text">
    <string>Manage Settings</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="menuResetSettings">
   <property name="text">
    <string>Reset Settings</string>
   </property>
  </action>
  <action name="menuHowItWorks">
   <property name="text">
    <string>How it Works</string>
   </property>
  </action>
  <action name="menuContactUs">
   <property name="text">
    <string>Contact us</string>
   </property>
  </action>
  <action name="menuAboutThisApp">
   <property name="text">
    <string>About this App</string>
   </property>
  </action>
  <action name="toolbarPresent">
   <property name="text">
    <string>PRESENT</string>
   </property>
  </action>
  <action name="toolbarEdit">
   <property name="text">
    <string>EDIT SONG</string>
   </property>
  </action>
  <action name="toolbarPrevious">
   <property name="text">
    <string>PREVIOUS</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="toolbarNext">
   <property name="text">
    <string>NEXT</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="toolbarBigger">
   <property name="text">
    <string>BIGGER</string>
   </property>
   <property name="shortcut">
    <string>=</string>
   </property>
  </action>
  <action name="toolbarSmaller">
   <property name="text">
    <string>SMALLER</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="toolbarBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BOLD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="toolbarSettings">
   <property name="text">
    <string>SETTINGS</string>
   </property>
  </action>
  <action name="actionDarkMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DARK MODE</string>
   </property>
  </action>
  <action name="menuDarkMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="menuSearchAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search All Song Books</string>
   </property>
   <property name="toolTip">
    <string>Search All Song Books</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
